docs/articles/expensify-classic/travel/Edit-or-cancel-travel-arrangements.md

```diff
@@ -3,9 +3,55 @@ title: Edit or cancel travel arrangements
 description: Modify travel arrangements booked with Expensify Travel
 ---
 <div id="expensify-classic" markdown="1">
+It's easy to change/cancel a flight booked with Expensify Travel. 
 
-Click **Get Support** on your emailed travel itinerary for real-time help with the booking. Any modifications, exchanges, or voidings made to a trip via support will incur a $25 booking change fee. 
+To modify or cancel a travel arrangement,
+<br>
+1. Navigate to Expensify Travel on an internet browser by clicking **Travel** and then **Book or Manage Travel**
+   ![image](https://github.com/Expensify/App/assets/3697042/c2d0959e-3428-489b-80ee-bfc3477fe96e)
```

Comment:
@ryanschaffer please follow the instructions [here](https://github.com/Expensify/App/tree/main/docs#add-images) to add images.

The URL is incorrect. It should be of this format - `![Image description here]({{site.url}}/assets/images/your_image_name_here.png){:width="100%"}`

```suggestion
   ![image](https://github.com/Expensify/App/assets/3697042/c2d0959e-3428-489b-80ee-bfc3477fe96e)
```